modules/microshift-greenboot-dir-structure.adoc

```diff
@@ -26,16 +26,16 @@ Returning a nonzero exit code from any script means that script has failed. Gree
 
 * `/etc/greenboot/check/required.d` contains the health checks that must not fail.
 
-** If the scripts fail, greenboot retries them three times by default. You can configure the number of retries in the `/etc/greenboot/greenboot.conf` file by setting the `GREENBOOT_MAX_BOOTS` parameter to the desired number of retries.
+** If the scripts fail, Greenboot retries them three times by default. You can configure the number of retries in the `/etc/greenboot/greenboot.conf` file by setting the `GREENBOOT_MAX_BOOTS` parameter to the desired number of retries.
```

Comment:
Am I safe in assuming there are no upper or lower restrictions in the number of retries? If there are restrictions, probably should mention them.

Reply:
That's a good point. The Fedora docs don't say.